packages/create-plugin/package.json

```diff
@@ -37,7 +37,7 @@
     "generate-datasource-backend": "tsc && npm run clean-generated && CREATE_PLUGIN_DEV=true node ./dist/bin/run.js --pluginName='Sample datasource' --orgName='sample-org' --pluginType='datasource' --hasBackend",
     "lint": "eslint --cache ./src",
     "lint:fix": "npm run lint -- --fix",
-    "test": "vitest",
+    "test": "vitest --passWithNoTests",
```

Comment:
can we instead make it ignore the particular example test?

Reply:
Thanks for pointing this out. I should have noted that it is temporary and will be removed when we add the first migration to the codebase. 

The issue isn't the example test it's that right now we have no migrations which causes the [migrations.test.ts](https://github.com/grafana/plugin-tools/blob/updates-as-migrations-pt1/packages/create-plugin/src/migrations/migrations.test.ts) that validates migrationScript paths are valid fails with `no test found in suite`. I can ignore that file temporarily but it felt like the lesser of two evils to use this flag instead given what "temporary" means. 😄